<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Interactive First Aid Cabinet</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1011</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="TITLE">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>50</y>
       <width>944</width>
       <height>76</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Unispace</family>
       <pointsize>48</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Scan QR Code to Continue</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="close_button">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover:!pressed
{
  background-color: rgb(195,30,60);
}

QPushButton
{

color: red;
border-style: outset;
border-width: 3px;
border-radius:10px;
border-color: black;
}
</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>348</y>
      <width>1001</width>
      <height>218</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="TUPC">
     <property name="geometry">
      <rect>
       <x>116</x>
       <y>13</y>
       <width>192</width>
       <height>192</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>192</width>
       <height>192</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>192</width>
       <height>192</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>pictures/program_pictures/tuplogo (1).png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="TUPC_2">
     <property name="geometry">
      <rect>
       <x>421</x>
       <y>12</y>
       <width>194</width>
       <height>194</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>194</width>
       <height>194</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>194</width>
       <height>194</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>pictures/program_pictures/green_cross.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="rasberry">
     <property name="geometry">
      <rect>
       <x>728</x>
       <y>9</y>
       <width>156</width>
       <height>200</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>156</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>156</width>
       <height>200</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>pictures/program_pictures/Raspberry logo.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="restart_button">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>170</y>
       <width>80</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-style: inset;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="hidden_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>80</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-style: inset;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="scan_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>471</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover:!pressed
{
  background-color: rgb(195,30,60);
}

QPushButton
{
  background-color: white;
color: black;
border-style: outset;
border-width: 3px;
border-radius:10px;
border-color: black;
}</string>
    </property>
    <property name="text">
     <string>SCAN NOW</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>600</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pictures/program_pictures/main_wallpaper.png</pixmap>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
   <zorder>scan_button</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
